_component_design/button-drop-down.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Drop-Down Buttons
```

Comment:
Should this be sentence case like other PRs I've seen today?

Reply:
Title case, which means that i need to update simple buttons and vertical stepper.